### ISSUE TITLE
FilterBar: missing accessibility properties for buttons action performed

### ISSUE THREAD

**Issue by sparkypro** [SAP/ui5-webcomponents-react#3124]
**Describe the bug**
Screen readers don't read the action performed by 'Hide Filter Bar' button. 

**UI Link**
Refer: https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-filterbar--default-story

**To Reproduce**
1. Click on the 'Hide Filter Bar' button
2. Observe Filters hide, then button changes to 'Show Filter Bar'
3. During this operation, for accessibility screen readers don't read out the action performed. Example 'Filter is hidden or shown'

**Expected behavior**
Accessibility must be enabled for action performed when 'Hide/Show Filter Bar' button click.

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:1.4.0
`@ui5/webcomponents-react` version:0.25.6
Operating System: Windows 11
Browser: Chrome

**Comment by sparkypro** [comment]
Retest with updated version, issue still persists.

**Comment by Lukas742** [comment]
We didn't find the time yet to look into this issue.
As soon as this is fixed the issue will be closed and a comment will be added once i's released.
